### ISSUE TITLE
Looks interesting but documentation is a bit light

### ISSUE THREAD

**Issue by toddheslin** [panoplyio/facebook-insight-stream#15]
This looks like a promising utility tool however I'm struggling to get it working. I suspect it was an internal project that was made public.

Is there any chance you can provide some insight (no pun intended) on which of the options are required and which are not? 

I'm specifically getting the error:
`TypeError: Object.keys(...).each is not a function`
On this line: https://github.com/panoplyio/facebook-insight-stream/blob/master/index.js#L350, which is being called here https://github.com/panoplyio/facebook-insight-stream/blob/master/index.js#L153

When I change it to `forEach()` it works fine. Not sure if there was another part of the code where `each` was added onto the Object prototype.

When I got that working, it seems we have a parsing error:
`SyntaxError: Unexpected token u in JSON at position 0 at FacebookInsightStream.parse (<anonymous>)`

At this point I'm thinking that something has been omitted from the project that I might need to continue. I also had the case where `npm install` didn't actually install any of the dependences as they weren't listed in the `package.json`.

I'd love to see what this app can do!

**Comment by kensaggy** [comment]
Hi Todd, thanks for reporting the issue! 

Indeed, like you suspected it was an internal project - so some dependencies were specified elsewhere.
We've updated the `package.json` to include the required dependencies as well updated the `README` to include an example `options` (See #16).

We've closed the issue for now as the dependency issue is resolved. If you're still having trouble feel free to open a new issue.
